src/services/feed.service.ts

```diff
@@ -0,0 +1,14 @@
+import { BasePostOfFeedDTO } from '@/dtos/feed.dto';
+import { PostEntity } from '@/entities';
+
+class FeedService {
+  public async getFeed(): Promise<BasePostOfFeedDTO[]> {
+    const feed: BasePostOfFeedDTO[] = (await PostEntity.find({
```

Comment:
This DTO doesn't really do anything for type safety. I think we leave it here for consistency and for us to understand what the expected return object should look like.